tests/testthat/test-xportr.R

```diff
@@ -0,0 +1,41 @@
+test_that("pipeline results match `xportr()` results", {
+  if (require(magrittr, quietly = TRUE)) {
+    test_dir <- tempdir()
+
+    pipeline_path <- file.path(test_dir, "adslpipe.xpt")
+    xportr_path <- file.path(test_dir, "adslxptr.xpt")
+
+    dataset_spec_low <- setNames(dataset_spec, tolower(names(dataset_spec)))
+    names(dataset_spec_low)[[2]] <- "label"
+
+    var_spec_low <- setNames(var_spec, tolower(names(var_spec)))
+    names(var_spec_low)[[5]] <- "type"
+
```

Comment:
Prevent messages from appearing in tests

```suggestion

  withr::local_message_sink(withr::local_tempfile())
```

Difference:

![image](https://github.com/atorus-research/xportr/assets/211358/829019ee-57fd-4457-b884-7b6d9a83bd07)